docs/sphinx/index.rst

```diff
@@ -70,15 +70,17 @@ SimEng was first presented by `Professor Simon McIntosh-Smith <http://uob-hpc.gi
 
 Additionally, other works concerning SimEng and its use can be found below:
 
+- Second International workshop on RISC-V for HPC - `An Empirical Comparison of the RISC-V and AArch64 Instruction Sets <https://dl.acm.org/doi/abs/10.1145/3624062.3624233>`_
```

Comment:
Can we add something to this to make the year clearly available.  Maybe "at SC2023"?  I think its good to immediately see that this work is current

Reply:
Sounds good